sympy/parsing/tests/test_sympy_parser.py

```diff
@@ -193,6 +190,16 @@ def test_recursive_evaluate_false_10560():
         assert parse_expr(text, evaluate=False) == parse_expr(result, evaluate=False)
 
 
+def test_issue__evaluateFalse_24288():
+    from sympy import symbols
+    x, y = symbols('x y')
+
+    assert parse_expr('x < y', evaluate=False) == Lt(x, y)
```

Comment:
These tests don't show what evaluate=False is dong because they pass either way:
```python
In [2]: parse_expr('x < y', evaluate=False)
Out[2]: x < y

In [3]: parse_expr('x < y')
Out[3]: x < y
```
We need something like this to show the effect of evaluate=False:
```python
In [1]: parse_expr('1 < 2', evaluate=False)
Out[1]: 1 < 2

In [4]: parse_expr('1 < 2')
Out[4]: True
```

Reply:
The result I got on my branch 
``` python
>>> type(parse_expr('1 < 2', evaluate=False))
<class 'sympy.core.relational.StrictLessThan'>
```
But `Lt(1, 2)` is boolean
but when I used non-numeric values I got the desired results
``` python 
>>> x,y = symbols('x y')
>>> type(Lt(x,y))
<class 'sympy.core.relational.StrictLessThan'>
```
If am not use the above results then how can I express the result in `<class 'sympy.core.relational.StrictLessThan'>`
Is their a way to express the results in `sympy.core.relational.StrictLessThan'>` in that form ?